oak-store-spi/pom.xml

```diff
@@ -26,7 +26,11 @@
             <instructions>
               <Export-Package>
                 org.apache.jackrabbit.oak.jcr,
-                org.apache.jackrabbit.oak.jcr.observation.filter
+                org.apache.jackrabbit.oak.jcr.observation.filter,
+                org.apache.jackrabbit.oak.json,
+                org.apache.jackrabbit.oak.spi.cluster,
+                org.apache.jackrabbit.oak.spi.commit,
+                org.apache.jackrabbit.oak.spi.state
```

Comment:
Do we export everything from oak-store-spi? If yes, just remove the whole `instructions` block and let the plugin export all packages. Otherwise, it might be simpler to remove individual packages using something like (untested)

```
<Export-Package>
*,
!org.apache.jacrakbbit.oak.blabla
</Export-Package>
```